ci.yml

```diff
@@ -1,7 +1,7 @@
 global:
         timeout: 2400
 script:
-        - oc login -u kubeadmin -p mhk2X-Y8ozE-9icYb-uLCdV --insecure-skip-tls-verify=true https://api.crc.testing:6443
+        - oc login -u kubeadmin -p sXrUX-mnUP4-oM4Lw-VbWUs --insecure-skip-tls-verify=true https://api.ciserver.lab.upshift.rdu2.redhat.com:6443 
```

Comment:
@pleimer Not sure why you committed these to my PR (this PR is never even expected to pass) but I'll assume you know what you're doing and aren't worried about merge conflicts.